src/EmbeddedManifold.jl

```diff
@@ -66,12 +70,14 @@ struct TransparentIsometricEmbedding <: AbstractIsometricEmbeddingType end
 """
     EmbeddedManifold{𝔽, MT <: Manifold, NT <: Manifold} <: AbstractDecoratorManifold{𝔽}
 
-A type to represent that a [`Manifold`](@ref) `M` of type `MT` is indeed an emebedded
-manifold and embedded into the manifold `N` of type `NT`.
-Points and tangent vectors are still represented in the representation of `M` and the
-manifold `M` is being decorated (contrary to [`AbstractEmbeddedManifold`](@ref)).
-To go to the representation in the embedding, the function [`embed`](@ref) should be used.
-[`project`](@ref) can be used to go the other way.
+A type to represent that a [`Manifold`](@ref) `M` of type `MT` can be embedded into a
```

Comment:
Maybe you could add that this type represents an explicit embedding of `M` into `N`? The phrase "can be embedded" is a bit vague.